apiv2/client.go

```diff
@@ -60,28 +61,30 @@ type Client interface {
 	systeminfo.Client
 	user.Client
 	webhook.Client
+	configurations.Client
 }
 
 // RESTClient implements the Client interface as a REST client
 type RESTClient struct {
-	auditlog    *auditlog.RESTClient
-	artifact    *artifact.RESTClient
-	gc          *gc.RESTClient
-	health      *health.RESTClient
-	label       *label.RESTClient
-	member      *member.RESTClient
-	project     *project.RESTClient
-	projectmeta *projectmeta.RESTClient
-	quota       *quota.RESTClient
-	registry    *registry.RESTClient
-	replication *replication.RESTClient
-	repository  *repository.RESTClient
-	retention   *retention.RESTClient
-	robot       *robot.RESTClient
-	robotv1     *robotv1.RESTClient
-	systeminfo  *systeminfo.RESTClient
-	user        *user.RESTClient
-	webhook     *webhook.RESTClient
+	auditlog      *auditlog.RESTClient
+	artifact      *artifact.RESTClient
+	gc            *gc.RESTClient
+	health        *health.RESTClient
+	label         *label.RESTClient
+	member        *member.RESTClient
+	project       *project.RESTClient
+	projectmeta   *projectmeta.RESTClient
+	quota         *quota.RESTClient
+	registry      *registry.RESTClient
+	replication   *replication.RESTClient
+	repository    *repository.RESTClient
+	retention     *retention.RESTClient
+	robot         *robot.RESTClient
+	robotv1       *robotv1.RESTClient
+	systeminfo    *systeminfo.RESTClient
+	user          *user.RESTClient
+	webhook       *webhook.RESTClient
+	configuration *configurations.RESTClient
```

Comment:
@Jeremy-boo Something i noticed just now: Renaming the new package to `configure` would be a lot more consistent with the other packages of the client (and with what's defined by [Goharbor's API definitions](https://demo.goharbor.io/devcenter-api-2.0)). For reference, take a look at the [`apiv2/internal/api/client`](https://github.com/mittwald/goharbor-client/tree/master/apiv2/internal/api/client) directory where `configure` is the resulting directory name after swagger generation.